plugins/main/windows/overlay/README.md

```diff
@@ -0,0 +1,27 @@
+# L2Bridge  plugin (Windows)
+
+## Overview
+
+With Overlay plugin, all containers (on the same host) are plugged into an Overlay network based on VXLAN encapsulation. 
+
+
+## Example configuration
+```
+{
+	"name": "mynet",
+	"type": "win-overlay",
```

Comment:
Need to keep the type consistent with the binary name, and to match win-bridge, let's rename the binary to "win-overlay".